src/Twig/RecordExtension.php

```diff
@@ -75,8 +75,12 @@ public function dummy_with_env(Environment $env, $input = null)
         return $input;
     }
 
-    public function pager(Environment $env, Pagerfanta $records, string $template = '_sub_pager.twig', string $class = 'pagination', string $theme = 'default', int $surround = 3)
+    public function pager(Environment $env, ?Pagerfanta $records, string $template = '_sub_pager.twig', string $class = 'pagination', string $theme = 'default', int $surround = 3)
```

Comment:
rename $env to $twig (everywhere in this class)

Reply:
I'll make a separate PR for that. See #368